### DISCUSSION TITLE
visual stimulus and patch clamp recording using bonsai

### DISCUSSION THREAD

**Discussion by sagi-idan** [bonsai-rx/bonsai#1267] [Q&A]
Hi everyone.
As part of my research project I am planning to do patch clamp recordings from V1 while presenting drifting gratings. I was wondering if bonsai and BonVision might be a good option to perform data acquisition and synchronization in a ms time scale.
Is there any reading material, or even better, an existing workflow that performs it and I can use?

All the best!
Sagi

**Comment by bruno-f-cruz** [comment]
https://bonvision.github.io/ 
Might be a good place to start!
